charts/dirk/templates/statefulset.yaml

```diff
@@ -80,10 +80,10 @@ spec:
           image: "{{ .Values.cliImage.repository }}:{{ .Values.cliImage.tag }}"
           imagePullPolicy: {{ .Values.cliImage.pullPolicy }}
           securityContext:
-            runAsNonRoot: {{ .Values.global.securityContext.runAsNonRoot | default true }}
-            runAsUser: {{ .Values.global.securityContext.runAsUser | default 10000 }}
-            fsGroup: {{ .Values.global.securityContext.fsGroup | default 10000 }}
-            readOnlyRootFilesystem: {{ .Values.global.securityContext.readOnlyRootFilesystem | default true }}
+            runAsNonRoot: {{ .Values.securityContext.runAsNonRoot | default true }}
```

Comment:
outdated, will not work. Should be taken from values.

Reply:
It is taken from values, but not from `global`.  Can you please expand? Would you like to revert and put the values under `global` instead?

Reply:
Yes, just use hardcode them in values instead. Like there: https://github.com/NethermindEth/charts/blob/0343615400b50affe86aa7a567bd2928319a76a4/charts/web3signer/values.yaml#L17

Reply:
Done [2e9225a](https://github.com/NethermindEth/charts/pull/91/commits/2e9225aaa9f5207c88126ec8d0fe09717e10e52e)